packages/frontend-v2/package.json

```diff
@@ -4,23 +4,24 @@
   "private": true,
   "scripts": {
     "dev": "next dev",
+    "prod": "next build && next start",
     "build": "next build",
     "start": "next start",
     "lint": "next lint"
   },
   "dependencies": {
-    "@chakra-ui/icons": "^2.0.8",
-    "@chakra-ui/react": "^1.8.8",
+    "@chakra-ui/icons": "^2.0.0",
+    "@chakra-ui/react": "^2.0.0",
     "@dnd-kit/core": "^6.0.5",
     "@emotion/react": "^11",
     "@emotion/styled": "^11",
     "@fontsource/montserrat-alternates": "^4.5.9",
     "@graduate/common": "workspace:*",
     "framer-motion": "^6",
     "immer": "^9.0.15",
-    "next": "12.1.4",
-    "react": "18.0.0",
-    "react-dom": "18.0.0",
+    "next": "latest",
+    "react": "latest",
+    "react-dom": "latest",
```

Comment:
Correct me if I'm wrong, but I think we might still want concrete version numbers here so we don't encounter surprise issues when React or Next get a major version update.

Reply:
definitely!! thank you for this!